java/ql/src/Security/CWE/CWE-90/LdapInjection.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name LDAP query built from user-controlled sources
+ * @description Building an LDAP query from user-controlled sources is vulnerable to insertion of
+ *              malicious LDAP code by the user.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id java/ldap-injection
+ * @tags security
+ *       external/cwe/cwe-090
+ */
+
+import semmle.code.java.Expr
```

Comment:
```suggestion
import java
```
It's generally preferred to start all java queries with `import java`.

Reply:
Ok, I'll change it.